projects/Mallard/src/helpers/files.ts

```diff
@@ -336,6 +338,9 @@ export const downloadAndUnzipIssue = async (
             'downloadBlocked',
             DownloadBlockedStatus[downloadBlocked],
         )
+        errorService.captureException(
+            new Error('Download Blocked: Required signal not available'),
```

Comment:
We can we pre/post-fix the `network type` with the error message (if that is not already available in sentry)

Reply:
I've added a ticket for that one. I think its needed everywhere